Userland/Utilities/pls.cpp

```diff
@@ -28,40 +28,46 @@ bool unveil_paths(const char*);
 // Unveil paths, given the current user's path and the command they want to execute
 bool unveil_paths(const char* command)
 {
-    bool did_unveil_ok = false;
+    auto trusted_path = "/bin:/usr/bin:/usr/local/bin";
+
+    // Unveil all trusted paths with browse permissions
+    for (auto directory : String { trusted_path }.split(':')) {
+        if (unveil(directory.characters(), "b") < 0) {
+            perror("unveil");
+            return 1;
+        }
+    }
```

Comment:
```suggestion
    // Unveil all trusted paths with browse permissions
    auto trusted_directories = Array { "/bin", "/usr/bin", "/usr/local/bin" };
    for (auto directory : trusted_directories) {
        if (unveil(directory, "b") < 0) {
            perror("unveil");
            return false;
        }
    }
```

Reply:
done. i had this originally defined at the top of the file. `trusted_paths` should eventually be abstracted away into a config flag.

Reply:
Are you sure? Your latest commit shows the old code for me. :)

Reply:
Fixed. I didn't notice you had also changed the directory parsing to an `Array`.

An `Array` makes more sense, for now.